webpages/styles/components/tooltips.css

```diff
@@ -17,7 +17,8 @@
   z-index: 10;
   left: 100%;
   margin-inline-start: 10px; /* distance from parent */
-  width: 200px;
+  width: max-content;
+  max-width: 200px;
```

Comment:
Most likely, this broke *something*

Reply:
Actually, this might be fine, but making all addon IDs a "tooltip" (and changing the HTML structure) broke the loading indicator and possibly more CSS selectors.

Reply:
That sounds pretty bad